src/components/StatsSection.tsx

```diff
@@ -1,5 +1,5 @@
 const links = [
-  { name: "Find a project", href: "#" },
+  { name: "Find a project", href: "/new-project" },
```

Comment:
Sorry for the confusion with the components (and its naming).
The proper link for **Find a project** is `/marketplace`